slave.mk

```diff
@@ -794,6 +795,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
         $$(addprefix $(TARGET_PATH)/,$$($$*_DOCKERS)) \
         $$(addprefix $(FILES_PATH)/,$$($$*_FILES)) \
 	$(if $(findstring y,$(ENABLE_ZTP)),$(addprefix $(IMAGE_DISTRO_DEBS_PATH)/,$(SONIC_ZTP))) \
+	$(if $(findstring y,$(INCLUDE_HOST_SERVICE)),$(addprefix $(IMAGE_DISTRO_DEBS_PATH)/,$(SONIC_HOST_SERVICE))) \
```

Comment:
There seems to be some tab/space issue. Can you please fix it?

Reply:
OK, I seem to have missed changing the ENABLE_HOST_SERVICE to INCLUDE_HOST_SERVICE in one file. Will do that and push the changes.

Reply:
I've also fixed the whitespace issue on these lines.